tests/asyncapi-2.0/Parameters Object/invalid-not-matching-param-name.yaml

```diff
@@ -0,0 +1,16 @@
+asyncapi: 2.0.0
+
+info:
+  title: Signup service example (internal)
+  version: 0.1.0
+
+channels:
+  user/{userId}/signup:
+    parameters:
+      userEmail:
+        schema:
+          type: string
+    publish:
+      message:
+        payload:
+          type: string
```

Comment:
But [this section says]:

> The key represents the name of the parameter. It MUST match the parameter name used in the parent channel name.

Reply:
Yes, we should probably clarify it there.

However, this test is the combination of [valid-extra-parameter.yaml](https://github.com/asyncapi/tck/blob/add_tck_tests/tests/asyncapi-2.0/Parameter%20Object/valid-extra-parameter.yaml) and [valid-parameter-not-defined.yaml](https://github.com/asyncapi/tck/blob/add_tck_tests/tests/asyncapi-2.0/Parameter%20Object/valid-parameter-not-defined.yaml).

This test should not be invalid. Or the [valid-parameter-not-defined.yaml](https://github.com/asyncapi/tck/blob/add_tck_tests/tests/asyncapi-2.0/Parameter%20Object/valid-parameter-not-defined.yaml) should be invalid too. I'd go for the first one. Parameters are not mandatory so I'd make this test valid or simply remove it.

Reply:
@derberg @jonaslagoni what do you think?

Reply:
I didn't to through this PR, are we here purely, I would say `0` `1`?
Or are there different severity levels, like also warnings and info? I'm asking because of course from spec point of view, this is not an error, you can have parameter in the channel name but it doesn't have to be defined. The thing is that it will cause some pain to the users https://github.com/asyncapi/parser-js/issues/24

So this is not invalid, same as `valid-parameter-not-defined.yaml` but would be awesome to share with user a warning message about this. As if it comes to the test, keep it, but mark as valid

Reply:
I'd say that's a valid/invalid test framework but @postatum can confirm.

Reply:
> Parameters are not mandatory so I'd make this test valid or simply remove it.

Ok, I'll just remove this test.

> I'd say that's a valid/invalid test framework but @postatum can confirm.

Yes, it's valid/invalid without severity levels.

Reply:
Awesome! I'm approving and merging it. Thanks, @postatum! Great stuff 💪